pandas/core/arrays/sparse/array.py

```diff
@@ -77,6 +81,17 @@
 
 import pandas.io.formats.printing as printing
 
+# See https://github.com/python/typing/issues/684
```

Comment:
why doing this here and not in pd._typing?

Reply:
This is the only place where we use `ellipsis` as a possible parameter to a method, which is why this construction is needed.  I can move it if you want.

Reply:
there's a TODO in pd._typing about adding Ellipsis to PositionalIndexer (also references the same GH issue as you did here)

Reply:
> there's a TODO in pd._typing about adding Ellipsis to PositionalIndexer (also references the same GH issue as you did here)

Fair enough, but I think we should handle `Ellipsis` in a separate PR, as I'm guessing there will be other things we have to fix if we add `Ellipsis` to `PositionalIndexer` in  `pd._typing`

Reply:
sure